integration_tests/tests/forseti_integration_testing/controls/inventory.rb

```diff
@@ -0,0 +1,493 @@
+require 'json'
+control 'inventory' do
+
+     describe "Create an inventory" do
+        subject do
+            command("forseti inventory list")
+        end
+
+        before do
+            command("forseti inventory create").result
+        end
+
+        its("stdout") { should match /SUCCESS/}
+
+        after do
+            command("forseti inventory purge 0").result
+        end
+    end
+
+    describe "Create an inventory and check database" do
+        subject do
+            command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select count(DISTINCT resource_id) from gcp_inventory where category='resource' and resource_type = 'project';\"")
+        end
+
+        before(:context) do
+            command("forseti inventory create").result
+            command("sudo apt-get -y install mysql-client").result
```

Comment:
Move the installation of `mysql-client` to the setup part.  Let's not have this installed in the test, which is being done multiple times.

Reply:
Done.

Reply:
@red2k18 with Forseti on GKE we are using the Python 3 implementation of MySQL; can we use that here as well? e.g. https://github.com/forseti-security/forseti-security/pull/3190